player/src/androidTest/kotlin/com/tidal/sdk/player/playlog/PlayLogTest.kt

```diff
@@ -347,7 +338,8 @@ class PlayLogTest {
             argThat {
                 with(gson.fromJson(this, JsonObject::class.java)["payload"].asJsonObject) {
                     assertThat(get("startAssetPosition").asDouble).isAssetPositionEqualTo(0.0)
-                    assertThat(get("endAssetPosition").asDouble).isAssetPositionEqualTo(5.065)
+                    assertThat(get("endAssetPosition").asDouble)
+                        .isAssetPositionEqualTo(MEDIA_PRODUCT_DURATION_1_SECONDS)
```

Comment:
This wasn't added here, but what does the `1` stand for here? It seems like it should be named `MEDIA_PRODUCT_1_DURATION_SECONDS ` instead?

Reply:
yep the name is wrong T.T However, since it's not added here and the next PR https://github.com/tidal-music/tidal-sdk-android/pull/35 happens to rename this can we go ahead as is anyway?

Reply:
Let's go ahead